src/components/floating-emojis/DOGConfetti.js

```diff
@@ -0,0 +1,147 @@
+import React, { useMemo } from 'react';
+import Animated from 'react-native-reanimated';
+import { View, Dimensions, StyleSheet } from 'react-native';
+import { Emoji } from '../text';
+import { ImgixImage } from '../images';
+import dogSunglasses from '@/assets/partnerships/dogSunglasses.png';
+
+const NUM_CONFETTI = 15;
+const CONFETTI_SIZE = 30;
+
+const createConfetti = () => {
+  const { width: screenWidth } = Dimensions.get('screen');
+
+  return [...new Array(NUM_CONFETTI)].map((_, i) => {
+    const clock = new Animated.Clock();
```

Comment:
We should try making it with Reanimated 2, we will eventually have to rewrite that to the new version when we will be approaching new arch switch Reanimated 3 doesn't support old API

Reply:
If this is just a temporary thing, we should be fine just removing it later or something